python/GangaDirac/Lib/Files/DiracFile.py

```diff
@@ -470,14 +472,13 @@ def accessURL(self, thisSE='', protocol=''):
         return a random one from all the replicas. Also use the specified protocol - if none then use 
         the default. 
         """
-        from GangaDirac.Lib.Backends.DiracUtils import getAccessURLs
         lfns = []
         if len(self.subfiles) == 0:
             lfns.append(self.lfn)
         else:
             for i in self.subfiles:
                 lfns.append(i.lfn)
-        return getAccessURLs(lfns, thisSE, protocol)
+        return DiracUtils.getAccessURLs(lfns, thisSE, protocol)
 
```

Comment:
This doesn't work `NameError: global name 'DiracUtils' is not defined` . What is the reasoning for this change?